src/base/components/DeviceProfile.js

```diff
@@ -17,6 +152,102 @@ class DeviveProfileTab extends React.Component {
              providerLink !=undefined ?<ProviderLink providerLink={providerLink}/>:null
             }
 
+          <div className="row">
+            <div className="control-left control-panel col-sm-6">
+              <h1><i className="material-icons">notifications_active</i>Alerts</h1>              
+              <ToolTipLite className="tooltip-b" content="Available to Alpha testers only"><button class={this.state.devicestatus=== "off" ?  "btn btn-danger btn-sm" : "btn btn-success btn-sm"} type="button" id="Mote" ref="Mote" value={this.state.devicestatus}  onClick={this.setConfig} data-loading-text="Loading ...">Turn {this.state.devicestatus} Zolertia</button></ToolTipLite>
+            </div>
+            <div className="control-right control-panel col-sm-6">
+            <h1><i className="material-icons">info</i>Radio Parameters</h1>
+                            
+
+            <div class=" row">
+            <ToolTipLite className="tooltip-b" content="Transmission power in dBm.">
+              <label for="colFormLabelSm" class="col-sm-8 ">TXLevel:</label>
+              
+              <select class="col-sm-4" onChange={this.setTxConfig} value={this.state.tx_level}>
+                <option value="7">7</option>
+                <option value="5">5</option>
+                <option value="3">3</option>
+                <option value="1">1</option>
+                <option value="0">0</option>
+                <option value="-1">-1</option>
+                <option value="-3">-3</option>
+                <option value="-5">-5</option>
+                <option value="-7">-7</option>
+                <option value="-9">-9</option>
+                <option value="-11">-11</option>
+                <option value="-13">-13</option>
+                <option value="-15">-15</option>
+                <option value="-24">-24</option> 
```

Comment:
Why it's hardcoded?

Reply:
Its just a template now, in future it will not be hardcoded...